docs-2.0/3.ngql-guide/7.general-query-statements/5.lookup.md

```diff
@@ -20,7 +20,7 @@ This topic applies to native nGQL only.
 
 - If the specified property is not indexed when using the `LOOKUP` statement, Nebula Graph randomly selects one of the available indexes.
 
-  For example, the tag `player` has two properties, `name` and `age`. Both the tag `player` itself and the property `name` have indexes, but the property  `age` has no indexes. When running `LOOKUP ON player WHERE player.age == 36 YIELD player.name;`, Nebula Graph randomly uses one of the indexes of the tag `player` and the property `age`.
+  For example, the tag `player` has two properties, `name` and `age`. Both the tag `player` itself and the property `age` have indexes, but the property  `name` has no indexes. When running `LOOKUP ON player WHERE player.age == 36 YIELD player.name;`, Nebula Graph randomly uses one of the indexes of the tag `player` and the property `age`.
```

Comment:
That's not the case. In Nebula Graph v2.0.1, if you run `LOOKUP ON player WHERE player.age == 36 YIELD player.name;`, you must have an index for the property age. But since 2.5.0, Nebula Graph can use indexes on name or player itself.
I noticed that the Chinese doc has also been modified like this, please double confirm the changes.